drivers/can/can_mcp251xfd.h

```diff
@@ -69,6 +69,7 @@ BUILD_ASSERT(MCP251XFD_TEF_FIFO_SIZE + MCP251XFD_TX_QUEUE_SIZE +
 #define MCP251XFD_MAX_INT_HANDLER_CALLS  10
 #define MCP251XFD_INT_HANDLER_SLEEP_USEC 10000
 
+#define MCP251XFD_RESET_DELAY_USEC 5000
```

Comment:
Is this value from a datasheet or just a random value that seems to work? Please add a code comment.

Reply:
I know you are not the original author, but you are submitting something for inclusion in the project and therefore responsible for the changes being proposed.